### PR TITLE
Change bemjson to bemtree at Toolbox page

### PR DESCRIPTION
**Pull request by AnyaTyu**
@vithar 

You used block "article" for styles at this page. Unfortunately block "article" can use only html-content. I have developed block "promo-article" for these needs. What do you think about this solution? I think we can use this block for such cases in future and this will be easy :)

### CODE REVIEW

File: blocks/promo/promo-article/promo-article.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{
+    padding: 0 0 0 111px;
+}
+
+.promo-article__heading
+{
+    font-family: 'OpenSans-SemiBold', Arial, sans-serif;
+    font-weight: 300;
+
+    font-size: 1.5em;
```
Comment (vithar) [inline blocks/promo/promo-article/promo-article.css]: codestyle

Comment (vithar) [reply blocks/promo/promo-article/promo-article.css]: Just use

```
font: 300 150% 'OpenSans-SemiBold', Arial, sans-serif;
```

File: blocks/promo/promo-article/promo-article.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.promo-article__heading .promo-article__link
+{
+    margin-left: 0;
+}
+
+.promo-article__link
+{
+    display: block;
```
Comment (vithar) [inline blocks/promo/promo-article/promo-article.css]: codestyle

**Comment by AnyaTyu** [comment]
Fixed. Thanks :)

**Pull request merged**